Formula/sqlite.rb

```diff
@@ -36,6 +37,12 @@ class Sqlite < Formula
     sha256 "991b40fe8b2799edc215f7260b890f14a833512c9d9896aa080891330ffe4052"
   end
 
+  resource "regexp" do
+    url "http://localhost/regexp.c", :using => :nounzip
```

Comment:
See item 2 in the PR body. This is not fully ready to merge until I get full feedback and a plan for dealing with item 2.